README.md

```diff
@@ -46,7 +46,7 @@ An image building DOLFINx, Basix, UFL and FFCx from source can be built using:
 
 ```bash
 cd docker
-docker build -f ./docker/Dockerfile -t local_lab_env .
+docker build -f ./Dockerfile -t local_lab_env .
```

Comment:
This instructs the user to build the outermost docker file, which is not required. I would suggest reverting this.

Reply:
It is duplicated. One of the following works.
```
cd docker
docker build -f ./Dockerfile -t local_lab_env .
```
```
docker build -f ./docker/Dockerfile -t local_lab_env .
```

Reply:
Right. I would then go for:
1. remove the cd
2. Use the original instruction


Reasoning behind this is that then the context of the Dockerfile is the root of the repo